data_steward/tools/consolidated_reports/run_consolidated_report.sh

```diff
@@ -52,11 +52,11 @@ then
 fi
 
 set -e
-# create a new environment in directory curation_env
-virtualenv curation_env
+# create a new environment in directory curation_venv
+virtualenv curation_venv
```

Comment:
This file still has the redundant app_id variable. Please add TODO if deferring absolute path resolution fix.